contrib/default.config

```diff
@@ -46,18 +46,27 @@ ST_LINUXBOOT_VARIANT=minimal
 # into the LinuxBoot initramfs.
 ST_SIGNING_ROOT="out/keys/example_keys/root.cert"
 
-# ST_HOST_CONFIG_LOCATION defines the location where the host configuration is
-# stored.
+# ST_HOST_CONFIG_MODE defines where the host configuration is stored.
```

Comment:
`ST_HOST_CONFIG_MODE defines where the host configuration is stored inside the stboot installation.`
(to avoid confusion with `ST_HOST_CONFIG_FILE`)

Reply:
sure, makes sense. ack